specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,289 @@
+[v1_enum]
+typedef enum COREWEBVIEW2_BROWSING_DATA_KIND {
+  /// Specifies data stored by the AppCache DOM feature.
+  COREWEBVIEW2_BROWSING_DATA_KIND_APP_CACHE = 1 << 0,
+
+  /// Specifies file systems data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_FILE_SYSTEMS = 1 << 1,
+
+  /// Specifies data stored by the IndexedDB DOM feature.
+  COREWEBVIEW2_BROWSING_DATA_KIND_INDEXED_DB = 1 << 2,
+
+  /// Specifies data stored by the localStorage DOM API.
+  COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_STORAGE = 1 << 3,
+
+  /// Specifies data stored by the Web SQL database DOM API.
+  COREWEBVIEW2_BROWSING_DATA_KIND_WEB_SQL = 1 << 4,
+
+  /// Specifies cache storage which stores the network requests
+  /// and responses. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_CACHE_STORAGE = 1 << 5,
+
+  /// Specifies DOM storage data. This browsing data kind is inclusive 
+  /// of COREWEBVIEW2_BROWSING_DATA_KIND_APP_CACHE, 
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_FILE_SYSTEMS,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_INDEXED_DB, 
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_STORAGE,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_WEB_SQL, 
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_CACHE_STORAGE.
+  COREWEBVIEW2_BROWSING_DATA_KIND_DOM_STORAGE = 
+      COREWEBVIEW2_BROWSING_DATA_KIND_APP_CACHE|
+      COREWEBVIEW2_BROWSING_DATA_KIND_FILE_SYSTEMS | 
+      COREWEBVIEW2_BROWSING_DATA_KIND_INDEXED_DB |
+      COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_STORAGE | 
+      COREWEBVIEW2_BROWSING_DATA_KIND_WEB_SQL |
+      COREWEBVIEW2_BROWSING_DATA_KIND_CACHE_STORAGE,
+
+  /// Specifies HTTP cookies data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_COOKIES = 1 << 6,
+
+  /// Specifies site data. This browsing data kind
+  /// is inclusive of COREWEBVIEW2_BROWSING_DATA_KIND_DOM_STORAGE and
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_COOKIES. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_SITE = COREWEBVIEW2_BROWSING_DATA_KIND_DOM_STORAGE | 
+      COREWEBVIEW2_BROWSING_DATA_KIND_COOKIES, 
+
+  /// Specifies content in the HTTP cache including images and other files. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_HTTP_CACHE = 1 << 7,
+
+  /// Specifies download history data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_DOWNLOAD_HISTORY = 1 << 8, 
+
+  /// Specifies general autofill form data. 
+  /// This excludes password information and includes information like: 
+  /// names, street and email addresses, phone numbers, and arbitrary input. 
+  /// This also includes payment data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_GENERAL_AUTOFILL = 1 << 9, 
+
+  /// Specifies password autosave data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_PASSWORD_AUTOSAVE = 1 << 10,
+
+  /// Specifies browsing history data. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_BROWSING_HISTORY = 1 << 11,
+
+  /// Specifies settings data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_SETTINGS = 1 << 12,
+
+  /// Specifies profile data that should be wiped to make it look like a new profile.
+  /// This does not delete account-scoped data like passwords but will remove access
```

Comment:
You have PASSWORD_AUTOSAVE listed in the values below, is this not deleting different passwords?